cmd/dep/testdata/godep/expected_import_output.txt

```diff
@@ -0,0 +1,6 @@
+Detected godep configuration files...
+Converting from Godeps.json ...
+  Using ^0.8.1 as initial constraint for imported dep github.com/sdboyer/deptest
+  Trying ^0.8.1 (3f4c3be) as initial lock for imported dep github.com/sdboyer/deptest
```

Comment:
This is just an FYI. In #715 I am working on fixing the version set in the lock, and once that goes in the godep importer should get that fix automatically. I'll make sure to verify that this test case is updated, as we shouldn't be locking to a version with an operator, i.e. it should be `Trying v0.8.1` instead of `Trying ^0.8.1` in the feedback. That was my mistake but should be fixed soon!